site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Getting the NGINX Plus Gateway Fabric Image with JWT
+description: "Use your NGINX Gateway Fabric subscription JWT token to get the NGINX Plus Gateway Fabric image from the F5 Docker registry."
+weight: 150
+doctypes: ["install"]
+toc: true
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+Follow the steps in this document to pull the NGINX Plus Gateway Fabric image from the F5 Docker registry into your Kubernetes cluster using your JWT token.
+
+{{<important>}}
+An NGINX Plus subscription certificate and key will not work with the F5 Docker registry.
+
+For NGINX Gateway Fabric, you must have an Connectivity Stack for Kubernetes subscription -- download the NGINX Plus Gateway Fabric (per instance) JWT access token from [MyF5](https://my.f5.com).
+
+To list the available image tags using the Docker registry API, you will also need to download the NGINX Plus Gateway Fabric (per instance) certificate (`nginx-repo.crt`) and the key (`nginx-repo.key`) from [MyF5](https://my.f5.com).{{</important>}}
+
+{{< note >}}
+You can also get the image using alternative methods:
+
+- You can use Docker to pull an NGINX Gateway Fabric image with NGINX Plus and push it to your private registry by following the ["Pulling the Gateway Fabric Image"]({{< relref "installation/ngf-images/pulling-ngf-image" >}}) documentation.
+- You can build an NGINX Gateway Fabric image by following the ["Information on how to build an Gateway Fabric image"]({{< relref "installation/ngf-images/building-the-images" >}}) documentation.
+
+{{< /note >}}
+
+## Before You Begin
+
+You will need the following information from [MyF5](https://my.f5.com) for these steps:
+
+- A JWT Access Token (per instance) for NGINX Gateway Fabric from an active NGINX Gateway Fabric subscription.
+- The certificate (`nginx-repo.crt`) and key (`nginx-repo.key`) for each NGINX Gateway Fabric instance, used to list the available image tags from the Docker registry API.
+
+## Prepare NGINX Gateway Fabric
+
+1. Log into the [MyF5 Portal](https://my.f5.com/), navigate to your subscription details, and download the relevant .cert, .key and .JWT files.
+1. Create a Kubernetes secret using the JWT token. You should use `cat` to view the contents of the JWT token and store the output for use in later steps.
+1. Ensure there are no additional characters or extra whitespace that might have been accidentally added. This will break authorization and prevent the NGINX Gateway Fabric image from being downloaded.
+1. Modify your deployment (manifest or Helm) to use the Kubernetes secret created in step 3.
+1. Deploy NGINX Gateway Fabric into your Kubernetes cluster and verify that the installation has been successful.
+
+## Using the JWT token in a Docker Config Secret
+
+1. Create a Kubernetes `docker-registry` secret type on the cluster, using the JWT token as the username and `none` for password (as the password is not used).  The name of the docker server is `private-registry.nginx.com`.
```

Comment:
Is this the real registry name? Is this to be replaced by a user registry, or the F5 registry?